packages/untp-test-suite/scripts/createConfigFile.js

```diff
@@ -0,0 +1,74 @@
+import fs from 'fs';
+import semver from 'semver';
+
+const SCHEMAS_PATH = 'src/schemas';
+const CONFIG_PATH = 'src/config';
+
+const getPath = (path) => {
+  const parentDir = process.cwd();
+  return `${parentDir}/${path}`;
+};
+
+const getSchemaPath = () => getPath(SCHEMAS_PATH);
+const getSchemaTypeName = () => {
+  return new Promise((resolve, reject) => {
+    fs.readdir(getSchemaPath(), (err, files) => {
+      if (err) {
+        console.error(`Error reading directory: ${err.toString()}`);
+        reject(err);
+      } else {
+        resolve(files);
+      }
+    });
+  });
+};
+
+const getLastestSchemaVersion = (type) => {
+  const schemaPath = getSchemaPath();
+  try {
+    const schemaVersions = fs.readdirSync(`${schemaPath}/${String(type)}`);
+    const latestVersion = semver.maxSatisfying(schemaVersions, '*');
+    return latestVersion;
+  } catch (err) {
+    console.error(err);
+    return '';
+  }
+};
+
+const mapTypesAndVersions = async () => {
+  try {
+    const types = await getSchemaTypeName();
+
+    const mapped = types.map((type) => ({
+      type: type,
+      version: getLastestSchemaVersion(type),
+      dataPath: '',
+    }));
+
+    return mapped;
+  } catch (err) {
+    console.error(err);
+    return [];
+  }
+};
+
+const getConfigPath = () => getPath(CONFIG_PATH);
+
+/**
+ * Create a config file with the latest schema versions
+ */
+const createConfigFile = async () => {
+  const config = await mapTypesAndVersions();
+  const configData = JSON.stringify(config, null, 2);
+  fs.writeFile(`${getConfigPath()}/credentials.json`, configData, (err) => {
+    if (err) {
+      console.error(err);
+    } else {
+      console.log('Config file created successfully');
+    }
+  });
+};
```

Comment:
I think we should use a TypeScript file (.ts) for the `createConfigFile` to leverage the benefits of TypeScript in reducing potential errors and bugs.